packages/blocks/src/__internal__/utils/query.ts

```diff
@@ -29,6 +29,9 @@ export function getBlockById<T extends ElementTagName>(
   return container.querySelector<T>(`[${ATTR}="${id}"]` as T);
 }
 
+/*
+Be cautious!! If BlockSuite is embedded in other container, MouseEvent.clientX, MouseEvent.clientY will change
```

Comment:
What is the meaning of `BlockSuite` here? Can you give an example? Because this function and MouseEvent are not related.

Reply:
I'd rather not add redundant comment here :)